src/ReturnTypes/StorageDynamicStaticMethodReturnTypeExtension.php

```diff
@@ -25,7 +25,7 @@ public function getClass(): string
 
     public function isStaticMethodSupported(MethodReflection $methodReflection): bool
     {
-        return $methodReflection->getName() === 'disk' || $methodReflection->getName() === 'drive';
+        return in_array($methodReflection->getName(), ['disk', 'drive', 'cloud']);
```

Comment:
```suggestion
        return in_array($methodReflection->getName(), ['disk', 'drive', 'cloud'], strict: true);
```

and we could import it: `use function in_array;`